fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2041,7 +2050,7 @@ public void testGetObjectGraphVariants() throws IOException {
 
     @Test
     public void testGetObjectGraph() throws IOException {
-        logger.trace("Entering testGetObjectGraph()...");
+        LOGGER.trace("Entering testGetObjectGraph()...");
```

Comment:
Yeah, this is confusing. There is `logger` from the parent object and `LOGGER` declared in this file (before I got here). I changed a few trying to figure out which should actually be there (since the pattern is that `LOGGER` is usually used).  I shouldn't have touched any of them and left it alone. Changing it back now. 

Someone should clean this up at some point, unless there's a good reason to have `logger` from parent and `LOGGER` in this class.